packages/web/src/store/sagas.ts

```diff
@@ -176,7 +176,7 @@ export default function* rootSaga() {
     toastSagas(),
     shareSoundToTikTokModalSagas(),
     smartCollectionPageSagas(),
-    searchUsersModalSagas(),
+    searchUsersSagas(),
```

Comment:
nit: I liked `searchUsersModalSagas` to differentiate from regular search

Reply:
I actually was just gonna push a commit that renames everything to omit 'Modal' lol. But I see your point.. tryna think of something else that better describes the functionality.. but if not will fall back to modal.

Reply:
you know what modal is fine.

Reply:
this actually would be an ideal case for not using sagas at all and adopting the new RTK hotness when that gets merged: https://github.com/AudiusProject/audius-client/pull/2828